meinberlin/apps/projects/api.py

```diff
@@ -20,34 +20,17 @@ def __init__(self, *args, **kwargs):
         self.now = now
 
     def get_queryset(self):
-        projects = (
-            Project.objects.filter(
-                Q(project_type="a4projects.Project")
-                | Q(project_type="meinberlin_bplan.Bplan")
-            )
-            .filter(
-                Q(access=Access.PUBLIC) | Q(access=Access.SEMIPUBLIC),
-                is_draft=False,
-                is_archived=False,
-            )
-            .order_by("created")
-            .select_related("administrative_district", "organisation")
-            .prefetch_related(
-                "moderators",
-                "plans",
-                "organisation__initiators",
-                "module_set__phase_set",
-            )
-        )
-        return projects
+        return get_project_queryset()
 
     def list(self, request, *args, **kwargs):
         statustype = ""
         if "status" in self.request.GET:
             statustype = self.request.GET["status"]
         data = cache.get("projects_" + statustype)
         if data is None:
-            queryset = self.filter_queryset(self.get_queryset())
+            queryset = cache.get("project_queryset")
```

Comment:
the old code uses `self.filter_queryset()` but we don't seem to use it here anymore?